README.md

```diff
@@ -23,6 +23,10 @@ Beluga is an extensible C++17 library with a ground-up implementation of the Mon
 - Emphasis on the prevention of regressions and facilitation of code improvements through test coverage.
 - Semi-automated benchmarks that can be used to validate different configurations.
 
+https://github.com/user-attachments/assets/59bca7ea-fe78-4460-b058-96a5c75d09ec
```

Comment:
@nahueespinosa is this the correct way to make the video show up among `Ekumen-OS/beluga/assets`?

Reply:
Not sure what you mean, but this looks fine by me.